components/director/internal/domain/application/service_test.go

```diff
@@ -1431,122 +1021,25 @@ func TestService_CreateFromTemplate(t *testing.T) {
 				assert.Contains(t, err.Error(), testCase.ExpectedErr.Error())
 			} else {
 				require.Nil(t, err)
-			}
-
-			mock.AssertExpectationsForObjects(t, appRepo, intSysRepo, scenariosSvc, uidSvc, bndlSvc, formationSvc)
-		})
-	}
-
-	t.Run("Returns error on loading tenant", func(t *testing.T) {
-		svc := application.NewService(nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, "", nil)
-		// WHEN
-		_, err := svc.Create(context.TODO(), model.ApplicationRegisterInput{})
-		assert.True(t, apperrors.IsCannotReadTenant(err))
-	})
-}
-
-func TestService_Upsert_TrustedUpsert(t *testing.T) {
-	// given
-	timestamp := time.Now()
-	testErr := errors.New("Test error")
-	Documents := []*model.DocumentInput{
-		{Title: "foo", Description: "test", FetchRequest: &model.FetchRequestInput{URL: "doc.foo.bar"}},
-		{Title: "bar", Description: "test"},
-	}
-	modelInput := model.ApplicationRegisterInput{
-		Name: "foo.bar-not",
-		Webhooks: []*model.WebhookInput{
-			{URL: stringPtr("test.foo.com")},
-			{URL: stringPtr("test.bar.com")},
-		},
-
-		Labels: map[string]interface{}{
-			"label":                "value",
-			"applicationType":      "test-app",
-			"ppmsProductVersionId": "1",
-		},
-		IntegrationSystemID: &intSysID,
-		BaseURL:             str.Ptr("http://test.com"),
-	}
-
-	modelInputWithoutApplicationType := model.ApplicationRegisterInput{
-		Name: "foo.bar-not",
-		Labels: map[string]interface{}{
-			"label":                "value",
-			"ppmsProductVersionId": "1",
-		},
-		IntegrationSystemID: &intSysID,
-		BaseURL:             str.Ptr("http://test.com"),
-	}
-
-	modelInputWithoutBaseURL := model.ApplicationRegisterInput{
-		Name: "foo.bar-not",
-		Labels: map[string]interface{}{
-			"label":                "value",
-			"applicationType":      "test-app-with-ppms",
-			"ppmsProductVersionId": "1",
-		},
-		IntegrationSystemID: &intSysID,
-		BaseURL:             str.Ptr(""),
-	}
-
-	modelInputWithInvalidBaseURL := model.ApplicationRegisterInput{
-		Name: "foo.bar-not",
-		Labels: map[string]interface{}{
-			"label":                "value",
-			"applicationType":      "test-app-with-ppms",
-			"ppmsProductVersionId": "1",
-		},
-		IntegrationSystemID: &intSysID,
-		BaseURL:             str.Ptr("123://localhost"),
-	}
+			}
 
-	modelInputWithInvalidPpmsProductVersion := model.ApplicationRegisterInput{
-		Name: "foo.bar-not",
-		Labels: map[string]interface{}{
-			"label":                "value",
-			"applicationType":      "test-app-with-ppms",
-			"ppmsProductVersionId": "2",
-		},
-		IntegrationSystemID: &intSysID,
-		BaseURL:             str.Ptr("123://localhost"),
+			mock.AssertExpectationsForObjects(t, appRepo, intSysRepo, uidSvc, bndlSvc, formationSvc)
```

Comment:
`appNameNormalizer` which is `*normalizer.DefaultNormalizator` is not `mock.assertExpectationser` and can't pass it to `mock.AssertExpectationsForObjects` because the `AssertExpectations` method is missing.